.gitignore

```
@@ -0,0 +1,2 @@
+.idea/
+.vscode/
```

`.DS_Store`도 추가 가능할까요? 맥 사용자들에게 유용할 것 같습니다. cc: @Bumsu-Yi 

```suggestion
.idea/
.vscode/
.DS_Store
```

---

넵넵!